src/Endpoints/LabelEndpoint.php

```diff
@@ -27,4 +27,17 @@ public function get(string $id): Label
 
         return $this->serializer->deserialize($response, Label::class);
     }
+
+    /**
+     * @param string $id
+     * @return mixed
+     * @throws AuthorizationFailedException
+     * @throws RequestFailedException
+     */
+    public function trackAndTrace(string $id)
+    {
+        $response = $this->client->request('/track-and-trace/' . $id, Client::METHOD_GET);
```

Comment:
sprtinf

Reply:
is GET niet default?

Reply:
Method is in de SDK nooit predefined. Had er wel over nagedacht om een default in te stellen, maar ik vind het expliciet opgeven duidelijker omdat je altijd ziet wat the method is. Zouden we eventueel nog kunnen verhelpen door post/GET methods te maken. als laag ertussen